src/utils/prompts/blob-set-prompt.ts

```diff
@@ -0,0 +1,18 @@
+import { chalk, log } from '../command-helpers.js'
+
+import { confirmPrompt } from './confirm-prompt.js'
+
+const generateBlobWarningMessage = (key: string, storeName: string): void => {
+  log()
+  log(`${chalk.redBright('Warning')}: The following blob key already exists in store ${chalk.cyan(storeName)}:`)
+  log()
+  log(`${chalk.bold(key)}`)
+  log()
+  log(`This operation will ${chalk.redBright('overwrite')} the existing value.`)
+  log(`${chalk.yellowBright('Notice')}: To overwrite without this warning, you can use the --force flag.`)
```

Comment:
There's a lot of places where we re-use the same messages. Can we extract all these messages out to a single location to make it easier to change the message for all these commands

Reply:
A pattern I like for this type of thing is doing something like 
```
const destructiveCommandMessages = {
  blobs: {
    exist: 'yadda yadda',
    delete: 'yadda yadda'
  },
  clone: {
    something: 'yadda yadda',
    somethingElse: 'yadda yadda'
  }
  general: {
     noWarning: `${chalk.yellowBright('Notice')}: To overwrite without this warning, you can use the --force flag.`
  }
}

log(`${destructiveCommandMessages.blobs.exists} ${chalk.cyan(storeName)}:`)
log(destructiveCommandMessages.general.noWarning)
```

Doesn't have to be exactly like this but basically throwing all your related messages into a single object to make it easier to manage the messaging for a specific feature. Also helps with reusability

Reply:
You can also make them functions if you want to pass in a variable. like 
```
const destructiveCommandMessages = {
  blobs: {
    exist: 'yadda yadda',
    delete: (key) => 'yadda yadda ${key}'
  },
```